e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/report-reconciliation/bank-utilisation-report-page.spec.js

```diff
@@ -0,0 +1,118 @@
+import {
+  CURRENCY,
+  FeeRecordEntityMockBuilder,
+  REPORT_NOT_RECEIVED,
+  RECONCILIATION_IN_PROGRESS,
+  UtilisationReportEntityMockBuilder,
+  toIsoMonthStamp,
+  getPreviousReportPeriodForBankScheduleByMonth,
+  FEE_RECORD_STATUS,
+  getFormattedReportPeriodWithLongMonth,
+} from '@ukef/dtfs2-common';
+import pages from '../../../pages';
+import USERS from '../../../../fixtures/users';
+import { NODE_TASKS } from '../../../../../../e2e-fixtures';
+
+const { utilisationReportPage } = pages;
+
+context('Bank utilisation report page', () => {
+  const SUBMISSION_MONTH = toIsoMonthStamp(new Date());
+  const BANK_ID = '961';
+  const REPORT_ID = 1;
+  const FEE_RECORD_ID_ONE = '11';
+  const FACILITY_ID_ONE = '11111111';
+  let bankName;
+  let reportPeriodString;
+
+  const aTfmFacilityFacilitySnapshot = () => ({
+    coverStartDate: new Date().getTime(),
+    coverEndDate: new Date().getTime(),
+    interestPercentage: 5,
+    dayCountBasis: 365,
+    coverPercentage: 80,
+    value: 100000,
+  });
+
+  const TFM_FACILITIES = [
+    {
+      facilitySnapshot: { ...aTfmFacilityFacilitySnapshot(), ukefFacilityId: FACILITY_ID_ONE },
+    },
+  ];
```

Comment:
Please can we use the `getMatchingTfmFacilitiesForFeeRecords` here also since we don't need to have all this level of detail in this test?

Reply:
done